### PR TITLE
first pass edits for May release

### PR DESCRIPTION
**Pull request by rosemarymarano**
In the following sections, please check/verify:
**Requirements**, other than K8s 1.7, 1.8, 1.9, and 1.10
**Restrictions**, removed dynamic domains but not sure about the others.
**Installation**, manual steps that are listed in this doc.

Where do you want to call out these features: support for dynamic clusters, certification of Apache HTTP Server and Voyager (HAProxy), certification of NFS for Persistent Volumes?
After the second paragraph, there is a list of bulleted items, preceded by this line, "This release of the Operator provides a number of features to assist with the management of WebLogic domains in a Kubernetes environment, including:"
If so, perhaps, you can provide some context about how these features assist with the management of WebLogic domains in a Kubernetes environment.

Do the following sections need to change to accommodate, "We need to document how to bring the entire domain down using script ‘delete-weblogic-domain-resources.sh’"
Shutting down a domain
Removing a domain
Removing the operator

### CODE REVIEW

**Comment by mriccell** [comment]
Requirements, other than K8s 1.7, 1.8, 1.9, and 1.10 - the exact version of 1.7 I believe is 1.75.

File: README.md [hunk 1]
```diff
@@ -158,14 +157,14 @@ Please refer to [Shutting down a domain](site/shutdown-domain.md) for informatio
 
 ## Load balancing with the Traefik Ingress controller
 
-The initial Technology Preview release of the operator supports only the Traefik load balancer/Ingress controller.  Support for other load balancers is planned in the future.
-Please refer to [Load balancing with Traefik](site/traefik.md) for information about current capabilities.
+This release of the operator supports only the Traefik load balancer/Ingress controller.  Support for other load balancers is planned in the future.
```
Comment (mriccell) [inline README.md]: Yes Traefik will not be the only load balancer we support.  Probably the sub title of this section should change to "Load balancing"
Have sub sections for Traefik, Voyager (Lily He will provide documentation), Apache (Dongbo will provide documentation)

File: README.md [hunk 2]
```diff
@@ -158,14 +157,14 @@ Please refer to [Shutting down a domain](site/shutdown-domain.md) for informatio
...
+Please refer to [Load balancing with Traefik](site/traefik.md) for information about the current capabilities.
 
 [comment]: # (Exporting operator logs to ELK.  The operator provides an option to export its log files to the ELK stack. Please refer to [ELK integration]site/elk.md for information about this capability.)
 
 ## Removing a domain
 
-To permanently remove a domain from a Kubernetes cluster, first shut down the domain using the instructions provided above in the section titled “Shutting down a domain”, then remove the persistent volume claim and the persistent volume using these commands:
+To permanently remove a domain from a Kubernetes cluster, first shut down the domain using the instructions provided in [Shutting down a domain](site/shutdown-domain.md), then remove the persistent volume claim and the persistent volume using these commands:
```
Comment (mriccell) [inline README.md]: The new script delete-weblogic-domain-resources.sh has the ability to delete a specific domain, all domains.  When the script is invoked it cleans all the custom resource definition for the domain. The script has nice documentation, if you need to have this documentation reviewed Tom Barnes should be able to help.

File: README.md [hunk 3]
```diff
@@ -4,17 +4,17 @@ Built with [Wercker](http://www.wercker.com)
...
 
-*	A mechanism to create a WebLogic domain on a Kubernetes persistent volume
-*	A mechanism to define a WebLogic domain as a Kubernetes resource (using a Kubernetes custom resource definition)
-*	The ability to automatically start servers based on declarative startup parameters and desired states
-*	The ability to automatically expose the WebLogic Server Administration Console outside the Kubernetes cluster (if desired)
-*	The ability to automatically expose T3 channels outside the Kubernetes domain (if desired)
-*	The ability to automatically expose HTTP paths on a WebLogic domain outside the Kubernetes domain with load balancing, and to update the load balancer when Managed Servers in the WebLogic domain are started or stopped
-*	The ability to scale a WebLogic domain by starting and stopping Managed Servers on demand, or by integrating with a REST API to initiate scaling based on WLDF, Prometheus/Grafana or other rules
+*	A mechanism to create a WebLogic domain on a Kubernetes persistent volume.
```
Comment (mriccell) [inline README.md]: Add new bullet -*  A mechanism to manage a WebLogic Configured or Dynamic cluster.

Comment (mriccell) [inline README.md]: A mechanism to create a WebLogic domain on a Kubernetes persistent volume.  This persistent volume can reside in NFS.

**Comment by rosemarymarano** [comment]
The doc states, "Kubernetes 1.7.5+, 1.8.0+, 1.9.0+, 1.10.0 (check with `kubectl version`)." Is that correct or does it need to be modified?

File: README.md (hunk 1 above)
Comment (rosemarymarano) [reply README.md]: Ok. I put in the subsections Traefik, Voyager, and Apache and left Voyager and Apache blank for others to fill in. The Traefik section has the existing one line: Please refer to Load balancing with Traefik for information about current capabilities.

File: README.md (hunk 2 above)
Comment (rosemarymarano) [reply README.md]: So, for Removing a domain, users do not have to first shut it down and then remove the persistent volume and persistent volume claim? Should I just point to the delete-weblogic-domain-resources.sh script? Where is it located (I don't see it in the site directory)?

Comment (rjeberhard) [reply README.md]: The delete script is kubernetes/delete-weblogic-domain-resources.sh

Comment (rosemarymarano) [reply README.md]: Ok. I'll make reference to it and ask Tom Barnes to review my changes.

**Comment by rosemarymarano** [comment]
I am not yet done with the changes required for the README.

**Pull request merged**